deployments/helm/sumologic-the-coffee-bar/values.yaml

```diff
@@ -248,7 +304,34 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
+affinity: {} 
+
+postgres_affinity: 
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app
+            operator: In
+            values:
+            - postgres-db
+        topologyKey: "kubernetes.io/hostname"
+        namespaces: ["warp001","warp002","warp003","warp004","warp005"]
+
+
+cashdesk_affinity: 
```

Comment:
Can't accept affinities based on custom namespaces. Please leave it as generic `cashdesk_affinity: {}` and apply needed changes in your `custom-values.yaml`